sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/LongPointData.java

```diff
@@ -30,6 +32,15 @@
 
   public static LongPointData create(
       long startEpochNanos, long epochNanos, Attributes attributes, long value) {
-    return new AutoValue_LongPointData(startEpochNanos, epochNanos, attributes, value);
+    return create(startEpochNanos, epochNanos, attributes, value, Collections.emptyList());
+  }
+
+  public static LongPointData create(
```

Comment:
javadoc needed here (and on the method above, while we're at it)

Reply:
done